### DISCUSSION TITLE
LiveCharts2 longevity and health

### DISCUSSION THREAD

**Discussion by spewmaker** [Live-Charts/LiveCharts2#1319] [General]
Firstly, thanks @beto-rodriguez for creating this awesome library.
I recently adopted the library in a critical project, my concern is -  ### What happens if this library is depreciated or stopped being maintained for XYZ reason. 
How can I insulate myself and the project from that scenario? Can I localize the Nuget Package and run it forever? 
### What are some solutions to my concern?
Please feel free to correct me if I am thinking about this from a naïve perspective.
Any thoughts are appreciated.

**Comment by beto-rodriguez** [comment]
Hello

The idea is to maintain the project for years, but as all open-source projects, it is a dare to maintain it, I am planning to also release a paid version so we can have a budget for the project and make it easier to keep it alive.

The good news is that the library is open source, so you can always take the code and fix any issue you face.

> Can I localize the Nuget Package and run it forever?

Any package that is published to NuGet, it is available for ever to everyone, those versions will never disappear, and the idea is to keep maintaining LiveCharts for a long period of time, LiveCharts2 is a huge effort and there are no plans to waste all that work.

**Comment by spewmaker** [comment]
Thank you for the blazingly fast reply! That is very reassuring to hear. Hopefully one day I will be able to contribute towards maintenance upkeep of the project. Also excited to learn more about the paid tier of the library.
Thanks for making our lives easier!
